source/website/src/views/Step3.vue

```diff
@@ -463,6 +484,21 @@ SPDX-License-Identifier: Apache-2.0
             this.columns.push(column_definition)
           }
         )
+        // add liveramp identifier
+        this.items.filter(x => x.column_type === 'LiveRamp')
```

Comment:
There can only be one live ramp id. Once someone selects a ramp id column type then the should not be able to assign another column that column type (i.e. grey out that menu option).

Reply:
Resolved